doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -91,3 +91,151 @@ HAFS Workflow Schematic
+    - Reduce radiation time step from 1800 to 720s (HFSB only).
+    - Update CO2 fix files.
+
+**Ocean/Wave Coupling:**
+    - MOM6 ocean model coupling (HFSA only).
+    - CMEPS with inline-CDEPS coupling (HFSA only).
+
+**Post-processing:**
+    - Upgrade GFDL Tracker.
+
+**Software Dependencies:**
+    - See `here <https://github.com/hafs-community/HAFS/blob/production/hafs.v2/versions/run.ver>`__
+
+**Output Changes:**
+    - See `here <https://docs.google.com/presentation/d/1dGWu_k-CdiX_ndaRe89iDznyNwkEJfjVTagvvEVZQaw/edit?usp=sharing>`__
+
+Computer Resource Information
+-----------------------------
+
+**Computation resource updates:**
+    - See `here <https://docs.google.com/presentation/d/1otBHAi3hfB1Vu5lk9bfjOE-GZYalZiu9wjnFlevQRQw/edit?usp=sharing>`__
+
+**List of the module versions used in HAFS:**
+    - See `here <https://github.com/hafs-community/HAFS/blob/production/hafs.v2/versions/run.ver>`__
+
+**Data retention for files in:** 
+    - ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsa.yyyymmdd/hh`` and ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsb.yyyymmdd/hh``
+        - **Disk space usage:** Increased from 145/140 GB (HFSAv1/HFSBv1) per cycle to ~295/195 GB (HFSAv2/HFSBv2).
+        - **HPSS disk requirement:** Increased from ~142/126 GB (HFSAv1/HFSBv1) to ~228/133 GB (HFSAv2/HFSBv2).
+        - **Preferred data retention in COM:** Files to be kept for 7 days.
+
+**Input Datasets**
+    - Global Forecast System (GFS, including GFS/GDAS) NetCDF format analysis for atmospheric initial condition, and 3-hourly GRIB2 files (up to 129 h) for atmospheric lateral boundary conditions and atmospheric forcing for oceanic and wave model components. Global RTOFS for oceanic initial and boundary conditions for the ocean model component. GFS/GDAS global wave analysis/forecast for HFSA wave component initial and lateral boundary conditions. OBSPROC observational data for HAFS data assimilation.
+
+**Input DCOM Dependencies**
+- The following scripts require input files from DCOM directory:
+    - ``./scripts/exhafs_obs_prep.sh``: to decode/dump TDR/HDOB/TEMPDROP data
+    - ``./scripts/exhafs_msg_check.py``: to check the message files are properly created by checking the original NHC/JTWC message files in the DCOM directory.
+    - ``./ush/setup_hurricane``: for SDM to set up hurricane message files by extracting info from the original NHC/JTWC message files in the DCOM directory.
+
+Pre-implementation Testing
+--------------------------
+
+All jobs should be tested as part of this implementation.
+
+**Does this change require a 30-day evaluation (for major implementations only)?**
+    No, but all changes have been tested successfully using available input as per the following list: `IT tests list <https://docs.google.com/presentation/d/1qttu6HhZC3I2yEWLIcsTM7BUGxGyevhtZMUU61kSyYI/edit?usp=sharing>`__
+
+**Who are the suggested evaluators?**
+    NHC, CPHC, and JTWC for model products and MAG for GEMPAK files.
```

Comment:
@BinLiu-NOAA Did NHC, CPHC, and JTWC evaluate for model products and MAG for GEMPAK files? I know that was suggested, but it would be good if we could state what actually happened if we know.

Reply:
Yes, the stakeholders (NHC, etc.) does evaluate the HAFS model products and MAG for GEMPAK files.